### ISSUE TITLE
Markers | when adding markers with different icon types at the same time, markers take only one icon type

### ISSUE THREAD

**Issue by Kumar-Monisankar-Basu** [liodali/osm_flutter#575]
```
void addMarker() async {
    await controller.setMarkerOfStaticPoint(
      id: "pickup",
      markerIcon: MarkerIcon(
        iconWidget: Image.asset(
          "assets/img/pickup_pin.png",
          width: 80,
          height: 80,
        ),
      ),
    );
    await controller.setMarkerOfStaticPoint(
      id: "dropoff",
      markerIcon: MarkerIcon(
        iconWidget: Image.asset(
          "assets/img/drop_pin.png",
          width: 80,
          height: 80,
        ),
      ),
    );
```
    await controller.setStaticPosition(
      [GeoPoint(latitude: 23.02756018230479, longitude: 72.5813197391731)],
      "pickup",
    );

    await controller.setStaticPosition(
      [GeoPoint(latitude: 23.02726396414328, longitude: 72.5851928489523)],
      "dropoff",
    );

    loadMapRoad();
  }

i gave 2 different icons for 2 markers, but all markers will be created with the last icon which is
drop_pin.png

**Comment by Kumar-Monisankar-Basu** [comment]
Hello @liodali have you started looking into it? Do you think it is doable?

**Comment by liodali** [comment]
i will check it tonight and make hotfix for it

**Comment by liodali** [comment]
did you face that on all platforms ?

**Comment by liodali** [comment]
did you try to await  `loadMapRoad()`; and call it before 
```
await controller.setStaticPosition(
  [GeoPoint(latitude: 23.02756018230479, longitude: 72.5813197391731)],
  "pickup",
);

await controller.setStaticPosition(
  [GeoPoint(latitude: 23.02726396414328, longitude: 72.5851928489523)],
  "dropoff",
);

```

**Comment by Kumar-Monisankar-Basu** [comment]
> did you try to await `loadMapRoad()`; and call it before
> 
> ```
> await controller.setStaticPosition(
>   [GeoPoint(latitude: 23.02756018230479, longitude: 72.5813197391731)],
>   "pickup",
> );
> 
> await controller.setStaticPosition(
>   [GeoPoint(latitude: 23.02726396414328, longitude: 72.5851928489523)],
>   "dropoff",
> );
> ```

Yes i try this but nothing happened... Please give me a solution of it, its urgent 🙏

**Comment by liodali** [comment]
i will make hotfix

**Comment by liodali** [comment]
new version will be available in mins `1.3.6`
